notebooks/COS/SplitTag/SplitTag.ipynb

```diff
@@ -0,0 +1,982 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Your code here:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dressed-planner",
+   "metadata": {},
+   "source": [
+    "<a id=extractSp></a>\n",
+    "# 3. Extracting spectra from the sub-exposures using `CalCOS`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fewer-newspaper",
+   "metadata": {},
+   "source": [
+    "While for most circumstances, `CalCOS` is best run on an association file, (a fits table which specifies a series of exposures to calibrate and combine into a spectrum,) the `CalCOS` pipeline can be run on individual `rawtag` or `corrtag` files. \n",
+    "\n",
+    "* More information on running `CalCOS` can be found in [Chapter 3.6 of the COS Instrument Handbook](https://hst-docs.stsci.edu/cosdhb/chapter-3-cos-calibration/3-6-customizing-cos-data-calibration#id-3.6CustomizingCOSDataCalibration-3.6.1MechanicsofTailoredRecalibration)\n",
+    "* An interactive walkthrough to running the `CalCOS` can be found in [our notebook on the `CalCOS` pipeline](https://github.com/spacetelescope/notebooks/blob/master/notebooks/COS/CalCOS/CalCOS.ipynb).\n",
+    "\n",
+    "To run `CalCOS`, your computer will need an `lref` system variable to tell the pipeline where to find your reference files. If you are on the STScI internet network (including via VPN), you may set this to the shared Institute lref directory. However, if you're not on the network, you will need to use the `CRDS` command to download the necessary reference files. This process is described in detail in Chapter 3 of our [notebook on setting up an environment for working with COS data](https://spacetelescope.github.io/COS-Notebooks/Setup.html#crdsS). \n",
+    "\n",
+    "Once you have downloaded the files, you can set the environment variable to your CRDS cache using the code cell below.\n",
+    "The cell attempts to set your `lref` variable to the correct directory, but if you have created your own lref cache, you will need to replace the `<PATH TO YOUR CRDS REFERENCE FILE DIR>` string with the appropriate path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cleared-tradition",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if not os.environ.get('lref'):\n",
+    "    if os.path.exists('/grp/hst/cdbs/lref/'): # IF on STScI Network/VPN\n",
+    "        os.environ['lref'] = '/grp/hst/cdbs/lref/' \n",
+    "        print(\"It looks like you are probably on the STScI network; setting lref to '/grp/hst/cdbs/lref/'\")\n",
+    "    else: # IF NOT on STScI Network/VPN\n",
+    "        os.environ['lref'] = '<PATH TO YOUR CRDS REFERENCE FILE DIR>' # EDIT THIS PATH\n",
+    "        if not os.path.exists(os.environ['lref']): # Check if that path exists\n",
+    "            print(\"Uh oh, I don't think that's a valid path. Deleting it.\")\n",
+    "            del os.environ['lref'] # delete this nonexistant path\n",
```

Comment:
it might be helpful to raise an error directly here so that anyone clicking "run all cells" will get an error at the place that they need to make changes, rather than in the following cell.

It also might be clearer to check if the path exists before assigning it to the environment variable instead of setting and then having to use the `del`.

Reply:
I added an assertion test statement that there path exists.

On the second point, I think the current way makes sense, as the user is supposed to fill in the <path> and be warned if it doesn't exist.